azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participant/grid/ParticipantGridCellViewModel.kt

```diff
@@ -87,9 +98,14 @@ internal class ParticipantGridCellViewModel(
     private fun getVideoStreamModel(
         cameraVideoStreamModel: VideoViewModel?,
         screenShareVideoStreamModel: VideoViewModel?,
+        isOnHold: Boolean,
```

Comment:
I tested in iOS, if the remote participant is onHold, the callingSDK will remove the videoStream for this remote participant. So typically, we don't need to update videoStream based on if the remote participant is onHold or not